packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -284,6 +285,7 @@ export async function getCommonConfig(wco: WebpackConfigOptions): Promise<Config
         removeLicenses: buildOptions.extractLicenses,
         advanced: buildOptions.buildOptimizer,
       }),
+      new TransferSizePlugin(),
```

Comment:
I think we should only add this if the it's a browser build. There is no reason why the transfer size should be displayed when building a server bundle.